app/api/search/artist/route.ts

```diff
@@ -0,0 +1,26 @@
+import { NextRequest } from "next/server";
+import { getArtists } from "@/lib/supabase/in_process_artists/getArtists";
+
+export async function GET(req: NextRequest) {
+  const query = req.nextUrl.searchParams.get("query") || "";
+  if (!query || query.trim().length === 0) {
+    return Response.json({ artist: null });
+  }
+  if (query.length > 100) {
+    return Response.json({ error: "Query too long" }, { status: 400 });
+  }
```

Comment:
Will this endpoint fail if the search query is longer than 100 characters?
What negative effects happen if I send a query with more than 100 characters?
- If none, please remove this check 🚮